schematic/store/synapse.py

```diff
@@ -1429,10 +1442,10 @@ def make_synapse_table(self,
                 #create list of columns:
                 cols = []
                 for col in table_to_load.columns:
-                    if col in column_type_dictionary:
-                        col_type = column_type_dictionary[col]['column_type']
-                        max_size = column_type_dictionary[col]['maximum_size']
-                        max_list_len = column_type_dictionary[col]['maximum_list_length']
+                    if col in table_schema_by_cname:
+                        col_type = table_schema_by_cname[col]['columnType']
+                        max_size = table_schema_by_cname[col]['maximumSize']
+                        max_list_len = 250
```

Comment:
This parses the column type dictionary appropriately.

Reply:
I understand how you're parsing the dictionaries now